components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -21,6 +21,21 @@ import {
 } from "../../../redux/assignmentsSlice";
 import ExpandableContainer from "../ExpandableContainer";
 
+const returnWrapStyling = (assignment: UserAssignmentSubmissionsData) => {
+  let wrapStyle = "";
+  if (assignment.coding_assignment.assignment_name.length >= 3) {
+    wrapStyle = "truncate";
+  }
+  return wrapStyle;
+};
+
+const returnParentStyling = (assignment: UserAssignmentSubmissionsData) => {
+  let parentStyle = "";
+  if (assignment.coding_assignment.assignment_name.length >= 8) {
+    parentStyle = "w-1/8";
+  } else return parentStyle;
+};
```

Comment:
if the condition is true this sets the parentStyle to 'w-1/8' but doesn't return anything; if it's false it will return a blank string

you should set this function to always return parentStyle - in the case the condition is true, it will return a parentStyle of "w-1/8"

you're doing this correctly with the returnWrapStyling function